pkgs/development/python-modules/rapidocr-onnxruntime/default.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  setuptools,
+  pyclipper,
+  opencv4,
+  numpy,
+  six,
+  shapely,
+  pyyaml,
+  pillow,
+  onnxruntime,
+}:
+let
+  version = "1.3.22";
+
+  src = fetchFromGitHub {
+    owner = "RapidAI";
+    repo = "RapidOCR";
+    rev = "v${version}";
+    hash = "sha256-8h4DFhnI9imr+bYQZdlrl8UKUdpwnGK+SGxLXSMmcag=";
+  };
+in
+buildPythonPackage {
+  pname = "rapidocr-onnxruntime";
+  inherit version src;
+  pyproject = true;
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pyclipper
+    opencv4
+    numpy
+    six
+    shapely
+    pyyaml
+    pillow
+    onnxruntime
+  ];
+
+  sourceRoot = "${src.name}/python";
+  patches = [ ./setup-py-patch.patch ];
+
+  postPatch = ''
+    substituteInPlace requirements_ort.txt --replace-fail "opencv_python" "opencv"
+    mv setup_onnxruntime.py setup.py
+  '';
+
+  pyImportCheck = [ "rapidocr_onnxruntime" ];
```

```suggestion
  pythonImportsCheck = [ "rapidocr_onnxruntime" ];
```
> pythonImportsCheck is set. This is still a good smoke test even if pytestCheckHook is set.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-contributing

---

Oh my bad. Sorry for the misleading information.